github/repos_pages.go

```diff
@@ -55,8 +54,7 @@ func (s *RepositoriesService) EnablePageSite(ctx context.Context, owner, repo st
 	}
 
 	// TODO: remove custom Accept header when this API fully launches.
-	acceptHeaders := []string{mediaTypeEnablePagesAPIPreview, mediaTypePagesPreview}
-	req.Header.Set("Accept", strings.Join(acceptHeaders, ","))
+	req.Header.Set("Accept", mediaTypeEnablePagesAPIPreview)
```

Comment:
I don't think this latest change needed to be made. From the GitHub API docs, it says this endpoint takes both custom media types, correct?  If so, your earlier change should not have been reverted.

I had asked that the `DisablePageSite` endpoint be changed, not this one.

Reply:
Sorry, I missed the details 😅. I had updated the PR